test/unit/org/apache/cassandra/tools/AuditLogViewerTest.java

```diff
@@ -61,6 +71,65 @@ public void tearDown() throws IOException
         }
     }
 
+    @Test
+    public void testNoArgs()
+    {
+        ToolRunner tool = runner.invokeTool(toolPath);
+        assertTrue(tool.getStdout(), tool.getStdout().contains("usage:"));
+        assertTrue(tool.getCleanedStderr(), tool.getCleanedStderr().contains("Audit log files directory path is a required argument."));
+        assertEquals(1, tool.getExitCode());
+    }
+
+    @Test
+    public void testMaybeChangeDocs()
+    {
+        // If you added, modified options or help, please update docs if necessary
+        ToolRunner tool = runner.invokeTool(toolPath, "-h");
+        assertEquals("34aea94756d4f6760b836dfbfa7bbf06", DigestUtils.md5Hex(tool.getStdout()));
+    }
+
+    @Test
+    public void testHelpArg()
```

Comment:
-1 here from me. I really want _docs_ to be a first class citizen concern regarding tooling. So rather than being 'an extra test line' I wanted to give it it's own method to highlight the importance of keeping help/docs updated. wdyt?